site/docs/extensions/index.md

```diff
@@ -23,6 +23,27 @@ A Substrait plan can reference one or more YAML files via URI for extension. In
 | Type Variation     | The name as defined on the type variation object.            |
 | Function Signature | In a specific YAML, if there is only one function implementation with a specific name, a extension type declaration can reference the function using either simple or compound references. Simple references are simply the name of the function (e.g. `add`). Compound references (e.g. `add:i8_i8`)are described below. |
 
+A YAML file can also reference types and type variations defined in another YAML file. To do this, it must declare the YAML file it depends on using a key-value pair in the `dependencies` key, where the value is the URI to the YAML file, and the key is a valid identifier that can then be used as an identifier-safe alias for the URI. This alias can then be used as a `.`-separated namespace prefix wherever a type class or type variation name is expected.
+
+For example, if the YAML file at `/extension_types.yaml` defines a type called `point`, a different YAML file can use the type in a function declaration as follows:
+
+```yaml
+dependencies:
+  ext: /extension_types.yaml
```

Comment:
Is this a valid URI?

Reply:
It is a valid URI reference which we tend to accept in implementations.  To make this a better example we could just append `file:`.

Reply:
I've made that change.